### ISSUE TITLE
Indentation issue in line 192 of models.py

### ISSUE THREAD

**Issue by puneeshkhanna** [benihime91/SIIM-COVID19-DETECTION-KAGGLE#2]
Simple fix to push.

       if self.cfg.MODEL.HEAD.MULTI_DROPOUT.USE:
            print("Multi-Sample Dropout is active")
            self.multi_drop = True
            self.dropout = nn.ModuleList()
            for _ in range(cfg.MODEL.HEAD.MULTI_DROPOUT.NUM):
                self.dropout.append(nn.Dropout2d(cfg.MODEL.HEAD.DROPOUT))
        else:
        self.multi_drop = False
